app/screens/RootNavigator.tsx

```diff
@@ -13,6 +14,10 @@ import { WalletNavigator } from './WalletNavigator/WalletNavigator'
 export function RootNavigator (): JSX.Element {
   const { wallets, isLoaded } = useWalletPersistenceContext()
 
+  useEffect(() => {
+    void WalletNotifications.register()
```

Comment:
why is there a void here?

Reply:
TO avoid floating promise warning we have added void